packages/orchestration/src/types.ts

```diff
@@ -11,3 +11,20 @@ export type * from './orchestration-api.js';
 export type * from './exos/cosmos-interchain-service.js';
 export type * from './exos/chain-hub.js';
 export type * from './vat-orchestration.js';
+
+/**
+ * ({@link ZCF})-like tools for use in {@link OrchestrationFlow}s.
+ *
+ * @interface
+ */
+export type ZcfTools<CT = Record<string, unknown>> = Pick<
+  ZCF<CT>,
+  'atomicRearrange' | 'assertUniqueKeyword'
+> & {
+  makeInvitation: <R, A = undefined>(
+    offerHandler: OfferHandler<ERef<R>, A>,
+    description: string,
+    customDetails?: object,
+    proposalShape?: Pattern,
+  ) => Promise<Invitation<R, A>>;
+};
```

Comment:
let's use an actual interface. This doesn't need any parameterization because `CT` (Custom Terms) will not be available.
```suggestion
export interface ZcfTools {
  assertUniqueKeyword: ZCF['assertUniqueKeyword'];
  atomicRearrange: ZCF['atomicRearrange'];
  makeInvitation: ZCF['makeInvitation'];
}
```

Reply:
Ah yeah, an explicit interface like this makes more sense for when we start deviating